git-lfs.yaml

```diff
@@ -36,6 +36,9 @@ pipeline:
 
       # mitigate CVE-2022-27191
       go get golang.org/x/crypto/ssh
+
+      # mitigate GHSA-vvpx-j8f3-3w6h
+      go get golang.org/x/net@v0.7.0
```

Comment:
Is there a risk to using the version here? Will this cause us to accidentally downgrade in the future?

Reply:
I added a comment to the version number to tell the next person to try removing all of these. You're right we'd risk downgrading, but IMO that's better than getting a potentially new, unpinned version of the package each time we build.